policy/modules/admin/aide.te

```diff
@@ -58,3 +58,7 @@ tunable_policy(`aide_mmap_files',`
 optional_policy(`
 	seutil_use_newrole_fds(aide_t)
 ')
+
+optional_policy(`
+	systemd_stream_connect_userdb(aide_t)
```

Comment:
I would for now not address the stream connects because chances are that this can be implemented in a more efficient and scalable way eventually.

But, and I an not 100% sure of this, I vaguely recall that this might or might not be proper..

Reply:
I'll keep it until we can determine the pattern and update it or revert if it is found to be improper.

Reply:
Nevermind, I'll take it out.